src/util/features.js

```diff
@@ -0,0 +1,53 @@
+// @flow
```

Comment:
For starters I would re-organize this file, as its dealing with both components and utility functions. You can create a features folder inside

src/components/feature-checking
|-- util.js // holds the isFeatureEnabled, getFeatureConfig
|-- FeatureProvider // exports <FeatureContext.Provider>
|-- FeatureFlag // renders <FeatureContext.Consumer>
|-- FeatureContext // exports React.createContext({});
|-- flowTypes.js // exports all needed types
|-- index.js // exports all the above